common/lib/xmodule/xmodule/course_metadata_utils.py

```diff
@@ -0,0 +1,210 @@
+    like to just pass course.display_name and course.url_name as arguments to
+    this function, we can't do so without breaking those tests.
+
+    Arguments:
+        course (CourseDescriptor|CourseOverview): descriptor or overview of
+            said course.
+    """
+    # TODO: Consider changing this to use something like xml.sax.saxutils.escape
+    return (
+        course.display_name if course.display_name is not None
+        else course.url_name.replace('_', ' ')
+    ).replace('<', '&lt;').replace('>', '&gt;')
+
+
+def number_for_course_location(location):
+    """
+    Given a course's block usage locator, returns the course's number.
+
+    This is a "number" in the sense of the "course numbers" that you see at
+    lots of universities. For example, given a course
+    "Intro to Computer Science" with the course key "edX/CS-101/2014", the
+    course number would be "CS-101"
+
+    Arguments:
+        location (BlockUsageLocator): The usage locator of the course in
+            question.
+    """
+    return location.course
+
+
+def has_course_started(start_date):
+    """
+    Given a course's start datetime, returns whether the current time's past it.
+
+    Arguments:
+        start_date (datetime): The start datetime of the course in question.
+    """
+    # TODO: This will throw if start_date is None... consider changing this behavior?
+    return datetime.now(UTC()) > start_date
+
+
+def has_course_ended(end_date):
+    """
+    Given a course's end datetime, returns whether
+        (a) it is not None, and
+        (b) the current time is past it.
+
+    Arguments:
+        end_date (datetime): The end datetime of the course in question.
+    """
+    return datetime.now(UTC()) > end_date if end_date is not None else False
+
+
+def course_start_date_is_default(start, advertised_start):
+    """
+    Returns whether a course's start date hasn't yet been set.
+
+    Arguments:
+        start (datetime): The start datetime of the course in question.
+        advertised_start (str): The advertised start date of the course
+            in question.
+    """
+    return advertised_start is None and start == DEFAULT_START_DATE
+
+
+def _datetime_to_string(date_time, format_string, strftime_localized):
+    """
+    Formats the given datetime with the given function and format string.
+
+    Adds UTC to the resulting string if the format is DATE_TIME or TIME.
+
+    Arguments:
+        date_time (datetime): the datetime to be formatted
+        format_string (str): the date format type, as passed to strftime
+        strftime_localized ((datetime, str) -> str): a nm localized string
+            formatting function
+    """
+    # TODO: Is manually appending UTC really the right thing to do here? What if date_time isn't UTC?
+    result = strftime_localized(date_time, format_string)
+    return (
+        result + u" UTC" if format_string in ['DATE_TIME', 'TIME']
```

Comment:
Nope. What I meant is that because that nothing currently uses LONG_DATE or TIME at the moment, we can make changes to how start_datetime_text handles those values without breaking anything.